core/node/base_token_adjuster/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "zksync_base_token_adjuster"
```

Comment:
Naming suggestion (for crates, structures, components, layers, etc):
`base_token_adjuster` -> `base_token_ratio_perserver`
`base_token_fetcher` -> `base_token_ratio_provider`

Still clunky, but, ugh, IMHO, less clunky.